src/pint/pintk/plk.py

```diff
@@ -29,19 +29,18 @@
     )
 
 
-# Where is this meant for? Maybe it belongs in application creation?
-# log.debug(
-#    "This should also show up. test click revert, turn params on and off, and prefit model"
-# )
+log.debug(
+    "This should also show up. test click revert, turn params on and off, and prefit model"
+)
 
 plotlabels = {
     "pre-fit": [
-        "Pre-fit residual",
+        r"Pre-fit residual ($\mu$s)",
```

Comment:
Does this mean that residuals are always in us now?

Reply:
Sorry - accidentally copied over old code! This should be fixed now - the y axis scales to us, ms, and s.